<launch>
  <rosparam file="$(find my_command_quadruped)/config/goals.yaml" command="load" />
  <node pkg="my_command_quadruped" type="param_access_node" name="param_access_node" output="screen"/>
</launch>
